### DISCUSSION TITLE
Don't send notification when DatabaseTableSizeCheck returns ok

### DISCUSSION THREAD

**Discussion by szhorvath** [spatie/laravel-health#133] [Ideas]
Hi,

Thank you for the great package. 

Could you change the `DatabaseTableSizeCheck` ok result to the following? This is the only check that sends a notification when everything is OK. We started ignoring the notifications because it always sent ok messages and missed the warnings and alerts.  It would be more consistent with the other checks as well.

```php
// Before
if ($tooBigTables->isEmpty()) {
    return $result->ok('Table sizes are ok');
}

// After
if ($tooBigTables->isEmpty()) {
    return $result->ok()->shortSummary('Table sizes are ok');
}
```

Many thanks for considering my request.

**Comment by freekmurze** [comment]
Feel free to create your own check with this logic 👍

**Reply by mstnorris** [reply]
@freekmurze , hello, and firstly, thank you for the really useful package. I am curious as to the reasoning behind this as I believe it is the only check to alert when it hasn't failed?
